traitsui/tests/editors/test_enum_editor.py

```diff
@@ -439,52 +306,45 @@ def test_radio_enum_editor_button_update(self):
 
     def test_radio_enum_editor_pick(self):
         enum_edit = EnumModel()
-
-        with reraise_exceptions(), \
-                self.setup_gui(enum_edit, get_view("custom")) as editor:
+        tester = UITester()
+        with tester.create_ui(enum_edit, dict(view=get_view("custom"))) as ui:
 
             self.assertEqual(enum_edit.value, "one")
 
-            # The layout is: one, three, four \n two
-            click_radio_button(editor.control, 3)
-            process_cascade_events()
+            radio_editor = tester.find_by_name(ui, "value")
+            radio_editor.locate(locator.Index(3)).perform(command.MouseClick())
 
+            # The layout is: one, three, four \n two
```

Comment:
This is incorrect! Indexing should work in the same order as the enum. See issue #1204